commandapi-core/src/test/kotlin/Examples.kt

```diff
@@ -1835,7 +1755,7 @@ getServer().addRecipe(emeraldSwordRecipe)
 // Safely override with the recipe we've defined
 val arguments = listOf<Argument<*>>(
     RecipeArgument("recipe").replaceSafeSuggestions(SafeSuggestions.suggest {
-        arrayOf(emeraldSwordRecipe, /* Other recipes here */)
+        arrayOf(emeraldSwordRecipe /* Other recipes here */)
```

Comment:
This comma was intentional. It shouldn't have been removed.

Reply:
I didn't even know I removed that xD Will add that back now